사용자가 할 일 추가 하는 코드 추가중

94CJG opened the pull request:
ㅁ사용자가 할 일을 추가 하고
ㅁ화면에 랜더링이 되고,
ㅁ데이터가 계속 남아 있는 상태를 만들고 싶다.

94CJG committed:
사용자가 할 일 추가 하는 코드 추가중

ㅁ사용자가 할 일을 추가 하고
ㅁ화면에 랜더링이 되고,
ㅁ데이터가 계속 남아 있는 상태를 만들고 싶다.

Kwakcena commented on App.js:
```diff
@@ -0,0 +1,49 @@
+    }
+    setSend((addList) => [todo, ...addList]);
+  };
+  console.log(todo);
```
불필요한 콘솔은 없애주쇼

Kwakcena commented on App.js:
```diff
@@ -0,0 +1,49 @@
+      {/* ToDolist 작성하는 곳 */}
+      <div>
+        <h2>add ToDo-List</h2>
+        <form onSubmit={onSubmit}>
+          <input
+            onChange={onChange}
+            value={todo}
+            id="list"
+            type="text"
+            placeholder="write your toDolist"
+          />
+        </form>
+      </div>
+      <button>Add To Do</button>
```
`button` 이 form 에 포함되어 있지 않은데?
버튼을 누르면 할 일이 추가 되고 있어 ?

94CJG replied:
리뷰 남겨주기 전에 하나 해봤는데 그거 한번만 봐줘!!

94CJG committed:
랜더링 됐을때 화면 출력

ㅁ 랜더링 했을때 화면에 출력 할 수 있음.
ㅁ li추가 되면서 리스트들이 블럭 구조를 원함 (p태그) 처럼!
ㅁ 계속해서 고민중

94CJG committed:
button 수정 및 {send} 를 ul 안에 넣음

form 밖에 있을 때 마우스로 클릭시 정상작동 안함
form 안으로 넣었을 때 마우스로 클릭시 정상작동 함
send 변수를 li를 통해 리스트 추가를 했지만.. 안됨 ..
